packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -233,7 +233,14 @@ export default function VisualEditor( { styles } ) {
 							</div>
 						) }
 						<RecursionProvider>
-							<BlockList __experimentalLayout={ layout } />
+							<BlockList
```

Comment:
I'm catching up on some layout/template editor things so I may miss tons of context: the effect of this seems that we'll show the block gap everywhere (front, site editor, post editor with template editor) except in the post editor when the theme doesn't have a `theme.json` file (classic theme). Is this reading correct?

Reply:
No, the idea of the change here is that the editor Dom should mimic the frontend and in the case of the "post editor", when we're in "template mode" we're showing the whole template meaning the canvas wrapper need to have the `wp-site-blocks` className like the frontend. 

Incidentally, this has the effect of making sure the newly added style targeting the top level items to apply the "gap" get applied properly in the template mode.